### DISCUSSION TITLE
Customizing markdown preview settings in mdRenderer.js

### DISCUSSION THREAD

**Discussion by amerkay** [strapi/strapi#7552] [Legacy Topics]
I'm trying to change the markdown preview settings to `breaks: true` in [mdRenderer.js](https://github.com/strapi/strapi/blob/35aafde2288e6858a9b1757cf2555a8561421140/packages/strapi-plugin-content-manager/admin/src/components/PreviewWysiwyg/utils/mdRenderer.js#L18). 

What's the best/cleanest way to do so?

Great work! Keep it up :)

**Comment by amerkay** [comment]
Oops, figured out! It's [in the docs](https://strapi.io/documentation/v3.x/admin-panel/customization.html#development-mode) 🤦

**Reply by derrickmehaffy** [reply]
Glad you figured it out :wink:
